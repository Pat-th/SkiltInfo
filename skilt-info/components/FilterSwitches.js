import React, {useState, useEffect} from 'react';
import {StyleSheet, Switch, Text, View} from 'react-native';

const FilterSwitches = props => {

    return (
        <View>
            <Text>Metadata</Text><Switch onValueChange={() => props.setMetadata(!props.metadata)} value={props.metadata}/>
            <Text>Skiltnummer</Text><Switch onValueChange={() => props.setSkiltnummer(!props.skiltnummer)} value={props.skiltnummer}/>
            <Text>Ansikt</Text><Switch onValueChange={() => props.setAnsikt(!props.ansikt)} value={props.ansikt}/>
        </View>
    )
};

const styles = StyleSheet.create({});
export default FilterSwitches;