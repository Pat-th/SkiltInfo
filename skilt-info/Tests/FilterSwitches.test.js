import React from "react";
import renderer from "react-test-renderer";

import FilterSwitches from "../components/FilterSwitches";

describe("<FilterSwitches />", () => {
    it("Has 53 children", () => {
        const tree = renderer.create(<FilterSwitches />).toJSON();
        expect(tree.children.length).toBe(53);
    });
    it('Renders correctly', () => {
        const tree = renderer.create(<FilterSwitches />).toJSON();
        expect(tree).toMatchSnapshot();
      });
   });