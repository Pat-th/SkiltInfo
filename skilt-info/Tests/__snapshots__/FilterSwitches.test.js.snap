// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterSwitches /> Renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Metadata
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Geometri
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Lokasjon
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Relasjoner
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View>
    <Text
      style={
        Object {
          "fontSize": 32,
        }
      }
    >
      Egenskaper:
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Skiltnummer
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Ansiktsside Rettet Mot
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Oppsettingsdato
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Punkttilknytning
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Geometri, punkt
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Tekst
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Plasseringskode
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Tosidig Plate Med Ulike Motiv
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Størrelse
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Høyde
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Bredde
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Skiltform
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Tekst- og Symbolhøyde
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Farge
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Belysning
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Folieklasse
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Klappskilt
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Vedtaksnummer
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Skiftet dato
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Arkivnummer
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Eier
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "flex": 0.8,
          "fontSize": 20,
          "marginLeft": 10,
        }
      }
    >
      Vedlikeholdsansvarlig
    </Text>
    <RCTSwitch
      accessibilityRole="button"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onTintColor="#e53938"
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "flex": 0.2,
          },
        ]
      }
      thumbTintColor="white"
      value={false}
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#a38e8c",
        "borderBottomWidth": 0.8,
        "marginBottom": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 3,
      }
    }
  />
</View>
`;
